app/Repositories/Eloquents/PostRepository.php

```diff
@@ -301,4 +301,17 @@ public function createPostsTags($tags, $postId)
 
         return true;
     }
+
+    public function updatePostsTags($tags, $post_id)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PostRepository.php</br>**Line**: 305</br>Variable "post_id" is not in valid camel caps format